forms-flow-web/src/components/Dashboard/StatusChart.js

```diff
@@ -16,15 +17,30 @@ const COLORS = [
 
 // label={renderCustomizedLabel}
 const ChartForm = React.memo((props) => {
-  const { submissionsStatusList, submissionData } = props;
+  const { submissionsStatusList, submissionData, submissionStatusCountLoader } = props;
   const {formVersions, formName, parentFormId} = submissionData;
-  formVersions.sort((item1,item2)=>{
-    let version1 = item1.version.replace("v","");
-    let version2 = item2.version.replace("v","");
-    return version1 > version2 ? 1 : -1;
-  });
+  // here checked 
+  const checkedFormVersions =  useMemo(()=> {
+    if(formVersions[formVersions.length - 1].version){
+      return formVersions;
+      }
+    return formVersions.map((i,index)=>i.version === null ? 
+           {...i, version:`v${index + 1}`} : i );
+  },[formVersions]);
 
-  const version = formVersions?.length && formVersions[formVersions.length - 1].version;
+  const sortedFormVersions = useMemo(()=>{
+    if(checkedFormVersions.length > 1){
+      return checkedFormVersions.sort((item1, item2)=>{
+        const version1 = +item1.version?.replace("v");
```

Comment:
Why is this replacing needed

Reply:
Also sort . Sort was coming from back-end right?

Reply:
okay replace don't needed I checked. and the sorting for sometimes formversion array will send without version order.